snippets/visualbasic/VS_Snippets_CLR_System/system.Console-INSERTTABS/VB/inserttabs.vb

```diff
@@ -17,16 +17,14 @@ Public Class InsertTabs
    Private Const usageText As String = "Usage: INSERTTABS inputfile.txt outputfile.txt"
    
    Public Shared Function Main(args() As String) As Integer
-      Dim writer As StreamWriter = Nothing
-
       If args.Length < 2 Then
          Console.WriteLine(usageText)
          Return 1
       End If
       
       Try
          ' Attempt to open output file.
-         writer = New StreamWriter(args(1))
+         Dim writer As New StreamWriter(args(1))
```

Comment:
This also doesn't compile because of the use of `writer` outside the `try` block. `Using` statements should be used to wrap the two stream instantiations.